python/lsst/afw/image/slicing.py

```diff
@@ -87,67 +87,40 @@ def handleNegativeIndex(index, size, origin, default):
     return index
 
 
-def makePointFromIndices(x, y, origin, bbox):
-    """Create a Point2I from an x, y pair, correctly handling negative indices.
+def makeBoxFromSlices(sliceArgs, bboxGetter):
```

Comment:
Since this now does what `interpretSliceArgs` used to do (I think?) could we rename it to `interpretSliceArgs` and come up with a new name for the new thing?  Or do we not need the thing called `interpretSliceArgs` anymore, since I don't see it used by `__getitem__` and `__setitem__` below?

In any case, it definitely shouldn't be called somethign `Box`- or `Slice`-specific, since it also works with single-pixel indexing arguments.

Reply:
I think that we need two different functions, but I'm open to changing the names.

Because most of the multiband classes slice arrays, `interpretSliceArgs` returns indices or slices, which is what the multiband classes need. However single band objects require the additional step of transforming those indices into either a `Box2I` or `Point2I`, so the new name was supposed to convey that this is used for stack primitives.

Do you have a suggestion for a better name?